torchvision/transforms/functional_tensor.py

```diff
@@ -466,8 +466,18 @@ def resize(
     # Define align_corners to avoid warnings
     align_corners = False if interpolation in ["bilinear", "bicubic"] else None
 
+    # This is a perf hack to avoid <pytorch-issue>
+    # We are transforming (1, 1, H, W) into (1, 2, H, W) to force to take channels_first path
+    do_perf_hack = False
```

Comment:
Sounds good to put things on prototype.
However, JIT is not passing, I'll be searching for a workaround.

Reply:
Unless we're in a rush, wouldn't it be better to wait for a proper solution in core? Something similar to https://github.com/pytorch/pytorch/issues/83840#issuecomment-1261052574

I agree that with the approaching release we may not want to risk too much

Reply:
We need it sooner than later but to avoid issues we will move it to prototype. It is likely there will be many workarounds like this to close the gaps between V1 and V2 in terms of speed and make them usable for the TorchMultimodal team. Most of them will be removed long before the API moved out of prototype.

Reply:
while moving the hack to prototype we have a slow down due to the fact that we cast non-float data to float (and copy), this makes input contiguous and we have to process twice more data as we expanded (N, 1->2, H, W).